director/accounts/migrations/0018_auto_20200426_1815.py

```diff
@@ -2,9 +2,9 @@
 
 from django.db import migrations, models
 
-ACCOUNT_MEMBER_NAME = "Account member"
-ACCOUNT_ADMIN_NAME = "Account admin"
-ACCOUNT_VIEWER_NAME = "Account viewer"
```

Comment:
Presumably we need to make these changes in the prod database. Is that right?

Reply:
Yes - that's true. I'm assuming that's a case of running the `make migrate-proddb` task? Is there an easy way to test the prod db locally?

Reply:
oh `run-prod`!

Reply:
In addition we'll need to:

- add the right `user` field to existing personal accounts
- rename those accounts by dropping the `s-personal-account` part

Reply:
That's another good question: 

Current personal accounts will need upgrading. That should just involve saving every user (which then triggers the code to name the account appropriately).

Is that something I should add to the migrations as well? i.e. I'll write a script that re-saves all users?

Reply:
Sure. That deals with my second bullet point, right? We can do the first one manually.

Reply:
It would take care of both at once I think. As soon as you have the account->user relation working, then saving the user should take care of the name as well. Let me test that one out..

Reply:
Resolved in 6ac4d21